### DISCUSSION TITLE
External control of opened state for MultISelect component

### DISCUSSION THREAD

**Discussion by ivkirill** [mantinedev/mantine#2435] [Feature requests]
HI!

MultiSelect has a great feature props as `creatable`, but the values, after creation can be modified.

I am trying to resolve this problem inside my project, like notion does. But I discover that I can't control opened state for internal `SelectPopover` component.

<img width="748" alt="image" src="https://user-images.githubusercontent.com/11794800/189930939-4fe50822-8327-4080-89f4-13aad9cba8dd.png">

Any chance to add a prop providing an external control for open state?

**Comment by rtivital** [comment]
Hi @ivkirill, this feature will not be added any time soon, and if that will be added it would probably be much better to built a custom component that handles notion-like tag picking. You can use Popover component for that.
